docs/en/core-services/Caching.md

```diff
@@ -204,70 +190,99 @@ A typical configuration will have three tiers for Memory, SSD, and HDD.
+- **LRUAnnotator**: Annotates the blocks based on least-recently-used order.
+**This is Alluxio's default annotator**.
+- **LRFUAnnotator**: Annotates the blocks based on least-recently-used and least-frequently-used orders with a
 configurable weight.
     - If the weight is completely biased toward least-recently-used, the behavior will be the same
-    as the LRUEvictor.
-    - The applicable configuration properties are `alluxio.worker.evictor.lrfu.step.factor` and
-    `alluxio.worker.evictor.lrfu.attenuation.factor` which can be found in
-    [configuration properties]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.evictor.lrfu.step.factor)
-- **PartialLRUEvictor** : Evicts based on a least-recently-used policy but will choose one configured
-storage directory on the local worker with maximum free space and only evict from that directory.
-
-The evictor utilized by workers is determined by the Alluxio property
-[`alluxio.worker.evictor.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.evictor.class).
+    as the LRUAnnotator.
+    - The applicable configuration properties are `alluxio.worker.block.annotator.lrfu.step.factor` and
+    `alluxio.worker.block.annotator.lrfu.attenuation.factor`.
+
+The annotator utilized by workers is determined by the Alluxio property
+[`alluxio.worker.block.annotator.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.block.annotator.class).
 The property should specify the fully qualified class name within the configuration. Currently
 available options are:
 
-- `alluxio.worker.block.evictor.LRUEvictor`
-- `alluxio.worker.block.evictor.GreedyEvictor`
-- `alluxio.worker.block.evictor.LRFUEvictor`
-- `alluxio.worker.block.evictor.PartialLRUEvictor`
+- `alluxio.worker.block.annotator.LRUAnnotator`
+- `alluxio.worker.block.annotator.LRFUAnnotator`
+
+#### Evictor Emulation
+The old eviction policies are now deprecated and Alluxio provided implementations are replaced with appropriate annotation policies.
+Configuring old Alluxio evictors will cause worker startup failure with `java.lang.ClassNotFoundException`.
+
+However, Alluxio supports emulation mode which annotates blocks based on custom evictor implementation. The emulation assumes the configured eviction policy creates
+an eviction plan based on some kind of order and works by regularly extracting this order to be used in annotation activities.
+
+The old evictor configurations should be changes as below. (Failing to change the lef-over configuration will cause class load exceptions as old evictor implementations are deleted.)
+- LRUEvictor -> LRUAnnotator
+- GreedyEvictor -> LRUAnnotator
+- PartialLRUEvictor -> LRUAnnotator
+- LRFUEvictor -> LRFUAnnotator
+
+### Tiered Storage Management
+As block allocation/eviction no longer enforces a particular tier for new writes, a new block could end up in any configured tier.
```

Comment:
does it mean we can now write a file larger than the top tier capacity? (which is not possible previously)
if the answer is yes, better to indicate this improvement here.